primitives/api/proc-macro/src/impl_runtime_apis.rs

```diff
@@ -398,19 +401,57 @@ fn extend_with_runtime_decl_path(mut trait_: Path) -> Path {
 	trait_
 }
 
+fn generate_dummy_trait_impl(initial: &ItemImpl, trait_api_ver: u64) -> ItemImpl {
```

Comment:
All suggestions how to make this function more readable are welcome :/

Reply:
The whole function is gone. As we spoke offline - no need for dummy implementations, we'll implement the versioned trait directly.